### PR TITLE
docs(timeseries-preview): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-590.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesPreview/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+Component has embedded logic for fetching timeseries data and latest datapoint available for it. To make it work properly, component
+has to be wrapped into `ClientSDKProvider` component with defined client SDK instance. Fetching logic
+also can be redefined via `retrieveTimeseries` and `retrieveLatestDatapoint` props to retrieve timeseries data and latest datapoint respectively
+
+### Basic usage:
+
+Imports:
+
+```jsx
```
Comment (mrtraser) [inline src/components/TimeseriesPreview/stories/basic.story.mdx]: `jsx` -> `typescript`

Comment (amevck) [reply src/components/TimeseriesPreview/stories/basic.story.mdx]: For now `typescript` is not working. Therefore all stories there including `jsx` for now.

Comment (mrtraser) [reply src/components/TimeseriesPreview/stories/basic.story.mdx]: but storybook v5.3.1 support typescript highlighting that's why I'm asking you 😄

Comment (amevck) [reply src/components/TimeseriesPreview/stories/basic.story.mdx]: ah it's working now :) sure I'll update :+1:

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-590.gearboxjs.preview.cogniteapp.com

**Pull request merged**